cli/core/findStalePods.go

```diff
@@ -253,26 +257,26 @@ func FindStaleEigenpods(ctx context.Context, eth *ethclient.Client, nodeUrl stri
 		return nil, err
 	}
 
-	allValidatorsWithIndices := utils.Map(_allValidators, func(validator *phase0.Validator, index uint64) ValidatorWithIndex {
-		return ValidatorWithIndex{
+	allValidatorsWithIndices := lo.Map(_allValidators, func(validator *phase0.Validator, index int) utils.ValidatorWithIndex {
+		return utils.ValidatorWithIndex{
 			Validator: validator,
-			Index:     index,
+			Index:     uint64(index),
 		}
 	})
 
 	// TODO(pectra): this logic changes after the pectra upgrade.
-	allSlashedValidators := utils.Filter(allValidatorsWithIndices, func(v ValidatorWithIndex) bool {
+	allSlashedValidators := lo.Filter(allValidatorsWithIndices, func(v utils.ValidatorWithIndex, _ int) bool {
 		if !v.Validator.Slashed {
 			return false // we only care about slashed validators.
 		}
-		if v.Validator.WithdrawalCredentials[0] != 1 {
+		if v.Validator.WithdrawalCredentials[0] != 1 && v.Validator.WithdrawalCredentials[0] != 2 {
```

Comment:
support for 0x2 credentials

Reply:
a function for this would be nice for future changes but nbd